src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -203,13 +203,10 @@ def is_device(dev):
     # use lsblk first, fall back to using stat
     TYPE = lsblk(dev).get('TYPE')
     if TYPE:
-        return TYPE == 'disk'
+        return TYPE == 'disk' or TYPE == 'loop'
 
     # fallback to stat
     return _stat_is_device(os.lstat(dev).st_mode)
-    if stat.S_ISBLK(os.lstat(dev)):
-        return True
-    return False
```

Comment:
Note to wanderers who follow here: this block is probably removed since it is dead code (due to the return statement above).